tests/data/dataset_readers/brown_corpus_dataset_reader_test.py

```diff
@@ -0,0 +1,23 @@
+# pylint: disable=no-self-use,invalid-name
+from allennlp.data.dataset_readers import BrownCorpusDatasetReader
+from allennlp.common.testing import AllenNlpTestCase
+
+
+class TestBrownCorpusDatasetReader(AllenNlpTestCase):
+    def test_read_from_file(self):
+        reader = BrownCorpusDatasetReader()
+        dataset = reader.read('tests/fixtures/data/brown_corpus.txt')
```

Comment:
If we re-use the sequence tagging reader instead of creating a new one, I'd still keep this fixture and this test, just make sure we can read the brown corpus with the right delimiter setting using the existing reader.